src/modules/commander/commander.cpp

```diff
@@ -2446,56 +2447,32 @@ Commander::run()
 			}
 		}
 
-		/* Check for failure combinations which lead to flight termination */
-		if (armed.armed &&
-		    !status_flags.circuit_breaker_flight_termination_disabled) {
-			/* At this point the data link and the gps system have been checked
-			 * If we are not in a manual (RC stick controlled mode)
-			 * and both failed we want to terminate the flight */
-			if (internal_state.main_state != commander_state_s::MAIN_STATE_MANUAL &&
```

Comment:
Did we lose the manual mode logic entirely?

Reply:
Normally, the logic for failsafe action is handled in [state_machine_helper.cpp](https://github.com/PX4/Firmware/blob/master/src/modules/commander/state_machine_helper.cpp). There are parameters in QGC to choose the failsafe action in case of data link loss and RC loss and should not be overwritten here.